Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/team_leader.yml

```
@@ -78,16 +78,19 @@
     - name: Clothing Items
       entries:
       - id: RMCScabbardMacheteFilled
+        name: machete scabbard (Full)
         points: 5
       - id: RMCPouchMacheteFilled
+        name: machete pouch (Full)
         points: 15
       - id: RMCBackpackRTO
         points: 5
-    #- id: CMWeldingGoggles
-    #  points: 3
+      - id: RMCWeldingGoggles
+        points: 3
```

This should wait until toggling them is in
```suggestion
      #- id: RMCWeldingGoggles
      #  points: 3
```